### PR TITLE
add Vector2.cross method

### PR DESCRIPTION
**Pull request by zhaoy-dev**
the 'cross product' of vector2 is very usefull though it's not well-defined!
https://msdn.microsoft.com/en-us/library/windows/desktop/microsoft.directx_sdk.geometric.xmvector2cross(v=vs.85).aspx

### CODE REVIEW

**Comment by yaoyao-cn** [comment]
i use this method to determine the rotation direction of two-finger touches event

**Comment by Mugen87** [comment]
Please also add the method to the doc page: https://threejs.org/docs/#api/math/Vector2

Besides, adding a unit test for this method would be great. You can use the [existing test](https://github.com/mrdoob/three.js/blob/943544960c63414dee8c6af8e2ed64df05e47b59/test/unit/src/math/Vector3.tests.js#L509-L520) for `Vector3` as an example.

File: docs/api/math/Vector2.html [hunk 1]
```diff
@@ -175,6 +175,12 @@ <h3>[method:Float dot]( [param:Vector2 v] )</h3>
 		<p>
 		Calculates the [link:https://en.wikipedia.org/wiki/Dot_product dot product] of this
 	  vector and [page:Vector2 v].
+        </p>
```
Comment (Mugen87) [inline docs/api/math/Vector2.html]: Please avoid mixing hard tabs with soft tabs. This will only mess up the formatting.

**Review by Mugen87 — CHANGES_REQUESTED** [review]

**Comment by Mugen87** [comment]
I think it also makes sense to add `.crossVectors()` and call this method inside of `.cross()`. In this way, `Vector2` is more consistent to `Vector3()`.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**